app/controllers/DeparturesController.scala

```diff
@@ -140,7 +137,8 @@ class DeparturesController @Inject()(cc: ControllerComponents,
         }
         .recover {
           case e =>
-            InternalServerError(s"Failed with the following error: $e")
+            Logger.error(s"Failed to create Departure with the following error: $e")
```

Comment:
This could be changed to `Logger.error(s"Failed to create departure", e)`. With this method, we log the stack trace.